addons/website_mass_mailing/static/src/js/website_mass_mailing.editor.js

```diff
@@ -64,6 +76,42 @@ options.registry.mailing_list_subscribe = options.Class.extend({
             self.getParent()._onRemoveClick($.Event( "click" ));
         });
     },
+    /**
+     * @override
+     */
+    cleanForSave() {
+        if (!this.$target[0].matches('.o_newsletter_popup')) {
```

Comment:
There is a dedicated class to handle the newsletter popup... so don't mention the newsletter popup class in the base class :grimacing:  Here I think you can just remove that `if`?